src/ui/wxWidgets/wxUtilities.cpp

```diff
@@ -637,3 +656,14 @@ void ImagePanel::DrawBitmapCentered(wxDC &dc, const wxSize &drawAreaSize, const
 
   dc.DrawBitmap(m_Bitmap, xCenterPosition, yCenterPosition, false);
 }
+
+wxTopLevelWindow* GetNearestTlvParent(wxWindow* win) {
```

Comment:
Maybe `GetNearestTopLevelParent`?  "tlv" is a bit obscure.

Reply:
`GetNearestTlvParent` is "dead code" from some experiments. Removed.